applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,103 @@
+.. _slm_as_zephyr_modem:
+
+nRF91 Series as a Zephyr-compatible modem
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+Starting with |NCS| v2.6.0, the Serial LTE Modem (SLM) application can be used to turn an nRF91 Series SiP into a standalone modem that can be used through Zephyr's cellular modem driver.
+This means that the controlling chip can run a Zephyr application that seamlessly uses Zephyr's IP stack instead of offloaded sockets via AT commands.
+
+This is made possible by SLM's support of CMUX and PPP and Zephyr's cellular modem driver.
+
+The :ref:`nRF52840 SoC present in the nRF9160 DK <zephyr:nrf9160dk_nrf52840>` can even be made to be the controlling chip with the appropriate configuration (described below).
+This is only possible on the nRF9160 DK, not on the other nRF91 Series DKs.
+
+.. note::
+
+   As of |NCS| v2.6, Zephyr's GNSS driver does not support the nRF91 Series running SLM.
+   This functionality is part of planned future work.
+
+Configuration
+*************
+
+Both SLM and the Zephyr application running on the controlling chip must be compiled with the appropriate configuration to make them work together.
+
+For that, Kconfig fragments and devicetree overlays must be added to the compilation command.
+
+See the :ref:`slm_config_files` section for a description of some of the mentioned Kconfig fragments.
+
+nRF91 Series SiP running SLM
+============================
+
+The following configuration files must be included:
+
+* :file:`overlay-cmux.conf` - To enable CMUX.
+* :file:`overlay-ppp.conf` - To enable PPP.
+* :file:`overlay-zephyr-modem.conf` - To tailor SLM to how Zephyr's cellular modem driver works.
+  This enables the :ref:`CONFIG_SLM_START_SLEEP <CONFIG_SLM_START_SLEEP>` Kconfig option, which makes the nRF91 Series SiP start only when the :ref:`power pin <CONFIG_SLM_POWER_PIN>` is toggled.
+
+In addition, if the controlling chip is an external MCU, the following files must also be included:
+
+* :file:`overlay-zephyr-modem-external-mcu.conf`- To define the power pin.
+  Make sure to update the defined pin so that it matches your setup.
+* :file:`overlay-external-mcu.overlay` - To configure the UART to be used.
+  Make sure to update its configuration (pins, baud rate) so that it matches your setup.
+  To update the configuration of the UART, you can directly modify the ``uart2`` node in the :file:`*_ns.overlay` overlay file matching your board in the :file:`boards` directory.
+
+Or, if the controlling chip is the nRF52840 of the nRF9160 DK, the following files must also be included:
+
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.conf` - To define the power pin.
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.overlay` - To configure the UART to be routed between the nRF9160 and the nRF52840 of the DK.
```

Comment:
Missing from: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/slm_description.html#configuration-files

Reply:
To answer all those `Missing from`, it was actually intended.
I thought that it doesn't make much sense to describe them there, and rather have them only here, in context, in the only place to which they are relevant: this page.
What do you think?

Reply:
I think that they should be at least mentioned in there. And linked to this page. So that the reader of that page understands the meaning of the overlays.

Reply:
Agreed they should be listed and linked to this for more details. Didn't notice they weren't listed there.